js/src/redux/providers/certifications/check-if-certified.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import ABI from '../../../contracts/abi/certifier.json';
+
+export default (api) => {
```

Comment:
If it makes sense, by all means.

This is not about personal taste, for the most part when you go into a component you cannot see if it was written by me, Nicolas, Jannis or Tomasz since it follows a consistent approach and way of doing things, where all adapted to. Would like to keep it that way and not leave personal preferences scattered along the codebase, I would prefer not looking at a component and knowing by the style who did it.